docs/reference/built-in-data-binding-converters.md

```diff
@@ -6,19 +6,27 @@ description: REFERENCE
-| `StringConverters.IsNotNullOrEmpty` | Returns `false` if the input string is null or empty                                                                                |
-| `ObjectConverters.IsNull`           | Returns `true` if the input is null                                                                                                 |
-| `ObjectConverters.IsNotNull`        | Returns `false` if the input is null                                                                                                |
-| `BoolConverters.And`                | A multi-value converter that returns `true` if all inputs are true.                                                                 |
-| `BoolConverters.Or`                 | A multi-value converter that returns `true` if any input is true.                                                                   |
+| Converter                           | Direction | Description                                                                                                |
+|-------------------------------------|-----------|------------------------------------------------------------------------------------------------------------|
+| Negation Operator                   | OneWay    | The ! operator can be placed in front of the data binding path to return the inversion of a Boolean value. |
+| `StringConverters.IsNullOrEmpty`    | OneWay    | Returns `true` if the input string is null or empty.                                                       |
+| `StringConverters.IsNotNullOrEmpty` | OneWay    | Returns `false` if the input string is null or empty.                                                      |
+| `ObjectConverters.IsNull`           | OneWay    | Returns `true` if the input is null.                                                                       |
+| `ObjectConverters.IsNotNull`        | OneWay    | Returns `false` if the input is null.                                                                      |
+| `BoolConverters.And`                | OneWay    | A multi-value converter that returns `true` if all inputs are true.                                        |
+| `BoolConverters.Or`                 | OneWay    | A multi-value converter that returns `true` if any input is true.                                          |
+| `BoolConverters.Not`                | OneWay    | Returns `true` if input is false, otherwise returns `false`. Similar to negation operator.                 |
+| `StringFormatValueConverter`        | OneWay    | Returns a formatted string with the supplied parameter.                                                    |
+| `StringFormatMultiValueConverter`   | OneWay    | Returns a formatted string with multiple input parameters.                                                 |
+| `EnumToBoolConverter`               | TwoWay    | Returns `true` if the input and parameter are the same, else `false`.                                      |
```

Comment:
In general, I would consider StringFormatValueConverter and StringFormatMultiValueConverter to be internal converters. That are actually used internally via Bindig.StringFormat.

EnumToBoolConverter probably could be changed to `EnumConverters.EqualsToParameter` or something.